docs/content/preview/explore/query-1-performance/yb-pg-stat-get-queries.md

```diff
@@ -0,0 +1,271 @@
+---
+title: View terminated queries with yb_pg_stat_get_queries
+linkTitle: View terminated queries
+description: Track planning and execution statistics for all SQL statements executed by a server.
+headerTitle: View terminated queries with yb_pg_stat_get_queries
+headcontent: See why a query failed
+menu:
+  preview:
+    identifier: yb-pg-stat-get-queries
+    parent: query-tuning
+    weight: 350
+type: docs
+---
+
+Use the YugabyteDB `yb_pg_stat_get_queries` view to see terminated queries and the reason for their termination.
```

Comment:
Use the YugabyteDB `yb_pg_stat_get_queries` **function** to see terminated queries and the reason for their termination.